CHARTER.adoc

```diff
@@ -1,9 +1,14 @@
+= Charter of The TODO Group
+
+:toc:
+
  . *Mission of the TODO Group (“TODO”).*
  The TODO Group’s mission is to identify key policy and process choices related to corporate open source engagement and create tools and educational materials that promote best practices around such engagements.
  . *Membership*
-The TODO Group membership is composed of individuals deeply involved in running open source programs setup to drive open source across divisions and product lines within their company. These individuals have learned from practical experience what it takes to successfully engage in or create open source projects and build effective internal open source programs that guide how developers in their company engage in external communities. +
+The TODO Group membership is composed of individuals deeply involved in running open source programs setup to drive open source across divisions and product lines within their organization. These individuals have learned from practical experience what it takes to successfully engage in or create open source projects and build effective internal open source programs that guide how developers in their company engage in external communities. +
 The TODO Group shall be composed of the General Members and a Steering Committee. Anyone may participate as a General Member as long as they meet the General Member Qualifications. At such time as a General Member no longer meets the qualifications, their General Member status will be rescinded.
  .. General Member Qualifications: An individual may be qualified to join TODO Group as a General Member if they:
+ ... Are an employee of an organization that is a member of The Linux Foundation at any membership level.
  ... Are an employee of an organization with development teams that use or contribute to open source projects across multiple divisions, products, or departments,
  ... in a role that manages, oversees, or guides the organization’s policies and contributions to open source projects (an “Open Source Program Office”),
  ... and the organization does not derive a significant portion of its revenue as a solution provider to other OSPOs. +
```

Comment:
on line 11: remove *TODO Group* ( they are free to participate in other community meetings in OSPOlogy, for instance). 

on line 12, change *company* for *organization*

Reply:
I am confused by your first point. There is no TODO Group mentioned in line 11.

Reply:
Perhaps add ..

... Are actively engaged as a project member contributing to and improving TODO's offerings, such as [perhaps noting scribe duties, hosting meetings, hosting TODO Group meetups]

Reply:
It is clear from the website (but not charter) that individual's can join the community as long as they abide by the code of conduct. https://todogroup.org/associates/

Reply:
@lhawthorn The charter only mentions general members and the steering committee. I suggest including a note at the beginning of the charter mentioning that this charter only covers general members and steering committee rights, criteria, etc, but the TODO Group structure covers more actors that are not included on the charter (OSPO associate organizations, individual contributors and OSPO practitioners, General member organizations and steering committee individuals)